### ISSUE TITLE
Google Drive Upload:  Cannot access a disposed object. Object name: 'System.Net.HttpWebResponse'

### ISSUE THREAD

**Issue by lsdh** [duplicati/duplicati#1696]
Tested on two computers, both are failing to upload properly to Google Drive account with 1TB of space.

```
Operation Put with file duplicati-b9d33d9a36d9842b5b9d16c256fa79917.dblock.zip.aes attempt 1 of 5 failed with message: Cannot access a disposed object. Object name: 'System.Net.HttpWebResponse'.
System.ObjectDisposedException: Cannot access a disposed object. Object name: 'System.Net.HttpWebResponse'. at System.Net.HttpWebResponse.CheckDisposed() at System.Net.HttpWebResponse.get_StatusCode() at Duplicati.Library.Backend.GoogleServices.GoogleCommon.ChunkedUpload[T](OAuthHelper oauth, String uploaduri, Stream stream) at Duplicati.Library.Backend.GoogleServices.GoogleCommon.ChunckedUploadWithResume[TRequest,TResponse](OAuthHelper oauth, TRequest requestdata, String url, Stream stream, String method) at Duplicati.Library.Backend.GoogleDrive.GoogleDrive.Put(String remotename, Stream stream) at Duplicati.Library.Main.BackendManager.DoPut(FileEntryItem item) at Duplicati.Library.Main.BackendManager.ThreadRun()
```

I have the full debug log if needed, but would prefer not to post it publicly.

**Comment by kenkendk** [comment]
The problem with this error message is that the `DisposedException` happens because something else has broken. This means that the message you see hides the real error. I doubt that there is more information in the debug log, but if there is, I would like to see it.

Is this on Windows?

**Comment by lsdh** [comment]
Hello,
Yes, this occurs on Windows. After further testing I have found the issue. It was a misconfiguration since I didn't understand one of the options. On the final setup page there is an option something to the effect of "Max Backup Size". By default it is 50MB. I was had thought that meant it wouldn't backup anything over 50MB of my hard drive, so I set this to 50GB. In turn, all backups fail. Setting this option back to default solved the issue.

Thanks!
